src/org/spdx/rdfparser/SpdxRdfConstants.java

```diff
@@ -53,7 +53,7 @@
 	public static final String RDFS_PROP_SEE_ALSO_DETAILS_WAYBACK_LINK = "isWayBackLink";
```

Comment:
2 suggestions:

- The naming convention for the constants is to use the `RSFS_` prefix for properties using the RDFS namespace.  There should not be any `RDFS_` prefix for these constants
- Since we are renaming the fields to be `CROSS_REF`... these should probably be renamed to `PROP_CROSS_REF_`...

Reply:
ok